src/client/containers/ContextMenuOptions.tsx

```diff
@@ -191,6 +197,12 @@ const NotesOptions: React.FC<NotesOptionsProps> = ({ clickedNote }) => {
         icon={Download}
         text={LabelText.DOWNLOAD}
       />
+      <ContextMenuOption
```

Comment:
The context menu should close after clicking this, that can be handled in another ticket though.